### DISCUSSION TITLE
"The Subuid and Subgid mapping has been performed, but the namespace separation is not occurring. What could be the reason?"

### DISCUSSION THREAD

**Discussion by mellowo** [containers/podman#21475] [General]
"I'm on CentOS 7 with Podman version 1.6.4. I performed the following mapping in /etc/subuid and /etc/subgid:

/etc/subuid:
myid:100000:65536

/etc/subgid:
myid:100000:65536

After that, I ran the 'podman system migrate' command. Following that, I checked whether the mapping was successful by running 'podman unshare cat /proc/self/uid_map', but it doesn't seem to change.

Could you provide any insights into why this might be happening?"

![podman](https://github.com/containers/podman/assets/27615279/0f457810-3139-4f93-8e06-faff4f1bb3b4)

**Comment by rhatdan** [comment]
I think on RHEL7 user namespace is not supported by default. You need to set a sysctl to turn it on.  BTW podman really is not supported on a system as old as RHEL7.
